src/main/java/emissary/client/response/PlacesResponseEntity.java

```diff
@@ -62,6 +67,6 @@ public void dumpToConsole() {
         for (PlaceList placeList : getCluster()) {
             placeList.dumpToConsole();
         }
-        System.out.println();
+        logger.info("");
```

Comment:
Do we still want an empty message in the logger?

Reply:
I don't think we use this output anywhere, but just in case, I wanted the output to be the same. I'm happy to remove it if you can confirm we don't use it anywhere.